src/main/python/cfn_sphere/exceptions.py

```diff
@@ -1,11 +1,24 @@
+from boto.exception import BotoServerError
+
+
 class CfnSphereException(Exception):
-    pass
+    def __init__(self, message="", boto_exception=None):
+        self.message = message
+        self.str = self.message
+        try:
+            self.request_id = boto_exception.request_id
+            self.str += " (Request ID: {0})".format(self.request_id)
+            self.boto_exception = boto_exception
+        except:
```

Comment:
@schlomo Silently capturing exceptions doesn't seem to be a good idea. Could you please handle the catched exceptions or remove the try-catch block?